examples/api300/thunderbird/unmanaged_device.rb

```diff
@@ -0,0 +1,45 @@
+# (C) Copyright 2016 Hewlett Packard Enterprise Development LP
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# You may not use this file except in compliance with the License.
+# You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed
+# under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+# CONDITIONS OF ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+
+require_relative '../../_client' # Gives access to @client
+
+def print_device(device)
+  puts "- \"#{device['name']}\" with model \"#{device['model']}\" and uri \"#{device['uri']}\""
+end
+
+puts "\nAdding first device ..."
+device1 = OneviewSDK::API300::Synergy::UnmanagedDevice.new(@client, name: 'Unmanaged Device 1', model: 'Procurve 4200VL')
+device1.add
+puts 'Device added:'
+print_device(device1)
```

Comment:
Change all instances of 
``print_device(device1)`` 
for something like
``puts "- \"#{device['name']}\" with model \"#{device['model']}\" and uri \"#{device['uri']}\""``